### PR TITLE
Use k8s secret for slack webhook url

### PR DESCRIPTION
**Pull request by luigibk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-1090

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: kured/templates/daemonset.yaml [hunk 1]
```diff
@@ -34,13 +36,18 @@ spec:
...
             - --slack-username={{ .Values.slackUsername }}
           env:
             - name: KURED_NODE_ID
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: SLACK_WEBHOOK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: kured-values
```
Comment (timwebster9) [inline kured/templates/daemonset.yaml:49]: should this be called something like `kured-secret` instead?

File: ci-values.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-image: quay.io/weaveworks/kured:support-k8s-1.10-5731b98 #updated image as default is not compatable with later v's of kubectl
+image: weaveworks/kured:1.1.0
 slackWebhookUrl:
```
Comment (timwebster9) [inline ci-values.yaml]: this can be removed now can't it?

File: kured/values.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-image: quay.io/weaveworks/kured:support-k8s-1.10-5731b98 #updated image as default is not compatable with later v's of kubectl
+image: weaveworks/kured:1.1.0
 slackWebhookUrl:
```
Comment (timwebster9) [inline kured/values.yaml]: same here (removed)

**Review by timwebster9 — COMMENTED** [review]
Did it not need anymore changes to the RBAC?

File: kured/templates/daemonset.yaml (hunk 1 above)
Comment (luigibk) [reply kured/templates/daemonset.yaml:49]: No, unless I change also the secret name in the other repo: https://github.com/hmcts/cnp-flux-config/blob/fd38f02669d62966a524dc2c71aac719e7469942/k8s/aat/kured/creds.yaml

**Comment by luigibk** [comment]
> Did it not need anymore changes to the RBAC?

I don't think so as we are not using RBAC.

**Comment by timwebster9** [comment]
> > Did it not need anymore changes to the RBAC?
> 
> I don't think so as we are not using RBAC.

we ARE using RBAC (this is k8s RBAC)

**Pull request closed without merging**